pkg/ccl/backupccl/restore_job.go

```diff
@@ -898,6 +898,8 @@ func createImportingDescriptors(
 
 			if eligible, err := backedUpDescriptorWithInProgressImportInto(ctx, p, desc); err != nil {
```

Comment:
[nit] maybe `s/eligible/isImporting` or something

Reply:
changed to `hasInProgressImportInto`. In progress IMPORT PGDUMP, for example, still gets skipped during regular restore, as that import command creates new table descriptors.